### PR TITLE
XAMARIN-44716-Prepared  UG content for RTE Move cursor to start or end position feature

### PR DESCRIPTION
**Pull request by HemalathaDuraisamy**
Hi Team,

Prepared  UG content for RTE Move cursor to start or end positionfeature

Regards,
Hemalatha D.

### CODE REVIEW

**Comment by HemalathaDuraisamy** [comment]
rebuild

File: Xamarin/Rich-Text-Editor/Basic-Features.md [hunk 1]
```diff
@@ -339,3 +339,37 @@ RichTextEditor.EnterKey = EnterKey.Div;
 {% endtabs %}
 
 N> This property works when loading the control only. It applies to iOS only and not supported in Android and UWP.
+
+## Move to the Cursor at Start or End Position
+
+Rich Text Editor provides the support to move the cursor position to the beginning or end of the content. By default, the cursor position is set at the end of the content.
```
Comment (Chrispineagunja) [inline Xamarin/Rich-Text-Editor/Basic-Features.md:345]: The Rich Text Editor provides support to move the cursor position to the beginning or end of the content. By default, the cursor position is set at the end of the content.

File: Xamarin/Rich-Text-Editor/Basic-Features.md [hunk 2]
```diff
@@ -339,3 +339,37 @@ RichTextEditor.EnterKey = EnterKey.Div;
...
+
+### MoveCursorToStart
+
+Rich Text Editor provides the support to move the cursor position to the beginning of the content. The following code example explains how to set 'MoveCursorToStart' mode in Rich Text Editor.
```
Comment (Chrispineagunja) [inline Xamarin/Rich-Text-Editor/Basic-Features.md:349]: The Rich Text Editor provides support to move the cursor position to the beginning of the content. The following code example explains setting the 'MoveCursorToStart' mode in the Rich Text Editor.

File: Xamarin/Rich-Text-Editor/Basic-Features.md [hunk 3]
```diff
@@ -339,3 +339,37 @@ RichTextEditor.EnterKey = EnterKey.Div;
...
+richtexteditor.VerticalOptions = LayoutOptions.FillAndExpand;
+richtexteditor.MoveCursorToStart();
+this.Content = richtexteditor;
+{% endhighlight %}
+
+{% endtabs %}
+
+### MoveCursorToEnd
+
+Rich Text Editor provides the support to move the cursor position to the end of the content. The following code example explains how to set 'MoveCursorToEnd' mode in Rich Text Editor.
```
Comment (Chrispineagunja) [inline Xamarin/Rich-Text-Editor/Basic-Features.md:364]: The Rich Text Editor provides support to move the cursor position to the end of the content. The following code example explains setting the 'MoveCursorToEnd' mode in the Rich Text Editor.

**Comment by HemalathaDuraisamy** [comment]
rebuild

**Review by Mohanaraja — APPROVED** [review]
changes are fine

**Pull request merged**